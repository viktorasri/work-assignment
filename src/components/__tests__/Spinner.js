import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../Spinner';

describe('<Spinner />', () => {
  const wrapper = shallow(<Spinner />);

  it('expect to render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
