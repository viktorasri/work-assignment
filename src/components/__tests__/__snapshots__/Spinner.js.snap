// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> expect to render component 1`] = `
<div
  className="spinner"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
